cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiImpl.java

```diff
@@ -443,14 +443,9 @@ public Future<Map<String, String>> listConnectLoggers(Reconciliation reconciliat
                                             Map<String, Map<String, String>> fetchedLoggers = mapper.readValue(buffer.getBytes(), MAP_OF_MAP_OF_STRINGS);
                                             Map<String, String> loggerMap = new HashMap<>(fetchedLoggers.size());
                                             for (var e : fetchedLoggers.entrySet()) {
-                                                if (Set.of("level", "last_modified").containsAll(e.getValue().keySet()))   {
-                                                    String level = e.getValue().get("level");
-                                                    if (level != null) {
-                                                        loggerMap.put(e.getKey(), level);
-                                                    }
-                                                } else {
-                                                    result.tryFail(new RuntimeException("Inner map has unexpected keys " + e.getValue().keySet()));
-                                                    break;
+                                                String level = e.getValue().get("level");
+                                                if (level != null) {
+                                                    loggerMap.put(e.getKey(), level);
                                                 }
```

Comment:
Was considering leaving the `tryFail` in case `level` doesn't exist at all, but noticed that no other operation on this class does this, so felt more natural to do it the same way.